### ISSUE TITLE
Missing Comment Likes Feature on Atomic

### ISSUE THREAD

**Issue by carinapilar** [Automattic/wp-calypso#52013]
### Steps to reproduce the behavior
1. On an Atomic site, leave a comment on a post.
2. With the comment likes feature enabled, leave some likes on the comment, using different users.

#### What I expected to happen
To show a link to open an overlay that lists all users that liked the comment, as it does on Simple sites:

![LikesOnSimple](https://user-images.githubusercontent.com/16104522/115012399-2ba56d00-9e86-11eb-8b38-bb967c5d7808.png)

#### What actually happened
This feature isn't available, there's no link, just plain text:

<img width="400" alt="LikesOnAT" src="https://user-images.githubusercontent.com/16104522/115012440-36600200-9e86-11eb-9b3d-7a5c92ea3a94.png">

### Context
It seems that this display-on-hover overlay is coming from the `comment-likes` must-use plugin. And we didn't find any similar functionality in Jetpack’s comment-likes module.

More details here: p9F6qB-6M2-p2

**Comment by brandonpayton** [comment]
I do not believe this should be implemented because WoA sites are completely under users' control, and it would be easy to take replace comment content with something offensive after many people have liked it. Then all the people who liked the original comment would erroneously appear to have personally endorsed the offensive content.

**Comment by brandonpayton** [comment]
> I do not believe this should be implemented because WoA sites are completely under users' control, and it would be easy to take replace comment content with something offensive after many people have liked it. Then all the people who liked the original comment would erroneously appear to have personally endorsed the offensive content.

I guess one counter argument would be that the same concern applies to posts, and we do at least show the avatars of people who've liked posts.

**Comment by dcoleonline** [comment]
Another report where a customer expected this to work as it did before they upgraded to Atomic
#4178909-zen

If we do update this, we should notify the customer.

**Comment by davipontesblog** [comment]
Changing priority to High as there is no workaround for this.
